tools/clang/lib/CodeGen/CGHLSLMS.cpp

```diff
@@ -6375,6 +6395,30 @@ Value *CGMSHLSLRuntime::EmitHLSLMatrixElement(CodeGenFunction &CGF,
   // -1 to avoid opcode param which is added in EmitHLSLMatrixOperationCallImp.
   Value *args[] = {paramList[HLOperandIndex::kMatSubscriptMatOpIdx - 1],
                    paramList[HLOperandIndex::kMatSubscriptSubOpIdx - 1]};
+
+  unsigned row, col;
```

Comment:
Please add a function, this is almost exactly the same code as your other change.

Reply:
Done.